README.md

```diff
@@ -1,9 +1,10 @@
 [![Discord](https://img.shields.io/badge/chat-on%20discord-7289da.svg?sanitize=true)](https://discord.gg/GUAy9wErNu)
 [![](https://img.shields.io/discord/908084610158714900)](https://discord.gg/GUAy9wErNu)
 [![Static Badge](https://img.shields.io/badge/github-assemblyline-blue?logo=github)](https://github.com/CybercentreCanada/assemblyline)
-[![Static Badge](https://img.shields.io/badge/github-assemblyline\_service\_ancestry-blue?logo=github)](https://github.com/CybercentreCanada/assemblyline-service-ancestry)
+[![Static Badge](https://img.shields.io/badge/github-assemblyline_service_ancestry-blue?logo=github)](https://github.com/CybercentreCanada/assemblyline-service-ancestry)
```

Comment:
Is this something that we should fix in the cuft template?

Reply:
If this is an issue with the cruft template, updating the cruft template and make all service just do a cruft update may be the reason we are using cruft? Unless it's only for initialization of a service, in which case most of the modifications here could have been skipped. I think we could use this example as a way to test updating the template and make sure it applies correctly everywhere.

Reply:
^ You can see the same mistake in the apivector migration.

Reply:
https://shields.io/badges#:~:text=The%20static%20badge%20accepts%20a%20single%20required%20path%20parameter%20which%20encodes%20either%3A